CK3toEU4/Data_Files/blankMod/output/common/cultures/!.expanded_culture_groups.txt

```diff
@@ -219,14 +230,6 @@ caucasian = {
 		primary = GEO
 		dynasty_names = { Andiyev Ashuraliyev Basiev Bazayev Bestayev Chibirov Din Dudarova Dzhanayev Fadzayev Farniev Gabulov Gassiev Gazyumov Guriev Khapov Khetagurov Kokoity Kusov Nyxas Pagayev Pliyeva Sanakoyev Tibilov Tokaty Torchinov Toshinori Valiyev Zangiev Zazirov }
 	}
-	assyrian = {
-		primary = GEO
-		dynasty_names = { Ahmadi Algafari Assadi Homsi Nasser }
-	}
-	coptic = {
-		primary = EGY
-		dynasty_names = { Thinis Memphis Abydos Herakleopolis Thebes Itjtawy Avaris Bubastis Sais }
-	}
```

Comment:
Why remove these cultures?

Reply:
I just discovered we have entries for them already in the other file.